console/cypress/integration/validators/validators.ts

```diff
@@ -36,8 +36,19 @@ export const createView = sql => {
 
 // ******************* VALIDATION FUNCTIONS *******************************
 
-// ******************* Remote Schema Validator ****************************
-export const validateRS = (remoteSchemaName, result) => {
+enum Result {
+  SUCCESS = 'success',
+  FAILURE = 'failure',
+}
+interface RequestBody {
+  [key: string]: any;
+}
+/**
+ * Remote Schema validator
```

Comment:
@beerose Let me know if we don't prefer jsDoc comments. Some might feel it is redundant info while working with TS but it would be helpful in non-ts file across the repo

Reply:
I'd get rid of them while migrating to TS.